packages/landingpage-new/components/home/core-aspects/core-aspects.module.scss

```diff
@@ -13,23 +13,18 @@
     flex-direction: column;
     align-items: center;
     justify-content: flex-start;
-    width: 300px;
     text-align: center;
     margin: auto 0;
-  }
-}
 
-@media only screen and (max-width: 1600px) {
-  .coreAspectsContainer {
-    .coreAspect {
-      width: 250px;
+    @media only screen and (max-width: 1600px) {
+      max-width: 200px;
```

Comment:
Could you change this `200px` in something like `14.286rem` or `15rem`?

In the [PR 945](https://github.com/inovex/elements/pull/945) I changed all `px` to `rem` and I think we should stay consistent with this unit throughout the landing page